subsys/bluetooth/controller/ll_sw/ull_sched.c

```diff
@@ -685,86 +789,96 @@ static void ticker_op_cb(uint32_t status, void *param)
 	*((uint32_t volatile *)param) = status;
 }
 
-#if defined(CONFIG_BT_CONN)
-static bool ticker_conn_match_op_cb(uint8_t ticker_id, uint32_t ticks_slot,
-				    uint32_t ticks_to_expire, void *op_context)
+static bool ticker_match_op_cb(uint8_t ticker_id, uint32_t ticks_slot,
+			       uint32_t ticks_to_expire, void *op_context)
 {
 	ARG_UNUSED(ticks_slot);
 	ARG_UNUSED(ticks_to_expire);
 	ARG_UNUSED(op_context);
 
-	return (ticker_id >= TICKER_ID_CONN_BASE) &&
-	       (ticker_id <= TICKER_ID_CONN_LAST);
-}
+	return false ||
 
-static struct ull_hdr *conn_ull_hdr_get_cb(uint8_t ticker_id)
-{
-	struct ll_conn *conn;
+#if defined(CONFIG_BT_CTLR_ADV_EXT) && defined(CONFIG_BT_BROADCASTER)
+	       IN_RANGE(ticker_id, TICKER_ID_ADV_AUX_BASE,
+			TICKER_ID_ADV_AUX_LAST) ||
 
-	conn = ll_conn_get(ticker_id - TICKER_ID_CONN_BASE);
-	if (!conn || conn->lll.role) {
-		return NULL;
-	}
+#if defined(CONFIG_BT_CTLR_ADV_PERIODIC)
+	       IN_RANGE(ticker_id, TICKER_ID_ADV_SYNC_BASE,
+			TICKER_ID_ADV_SYNC_LAST) ||
+#endif /* CONFIG_BT_CTLR_ADV_PERIODIC */
+#endif /* CONFIG_BT_CTLR_ADV_EXT && CONFIG_BT_BROADCASTER */
 
-	return &conn->ull;
+#if defined(CONFIG_BT_CONN)
+	       IN_RANGE(ticker_id, TICKER_ID_CONN_BASE,
+			TICKER_ID_CONN_LAST) ||
+#endif /* CONFIG_BT_CONN */
+
+	       false;
 }
 
-static void after_cen_offset_get(uint16_t conn_interval, uint32_t ticks_slot,
-				 uint32_t ticks_anchor,
-				 uint32_t *win_offset_us)
+static struct ull_hdr *ull_hdr_get_cb(uint8_t ticker_id, uint32_t *ticks_slot)
 {
-	uint32_t ticks_anchor_offset = ticks_anchor;
-	int err;
+	if (false) {
 
-	err = ull_sched_after_cen_slot_get(TICKER_USER_ID_ULL_LOW, ticks_slot,
-					   &ticks_anchor_offset,
-					   win_offset_us);
-	if (err) {
-		return;
-	}
+#if defined(CONFIG_BT_CTLR_ADV_EXT) && defined(CONFIG_BT_BROADCASTER)
```

Comment:
Can't we use `IS_ENABLED` here?

Reply:
`TICKER_ID_ADV_AUX_BASE` and `TICKER_ID_ADV_AUX_LAST` are not defined unless `CONFIG_BT_CTLR_ADV_EXT`